content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -22,21 +22,130 @@ external components communicate with one another.
+write applications using the Kubernetes API. 
+
+Kubernetes offers two "machine readable" APIs for listing all other Kubernetes
+APIs. Both are valuable tools for developers who are building client libraries,
+tooling, IDE features, documentation, and other tools interacting with
+Kubernetes.
+
+[Discovery](#discovery-api) is an API that provides information about available API names,
+resources, versions, and operations supported. This is a Kubernetes specific term. It is intended to be a small
+document providing a summary of available resources and does not detail specific
+schema for the resources. For reference around schemas for resources, please
+refer to the OpenAPI document.
+
+The [OpenAPI]{#openapi-specification} provides full [OpenAPI v2.0 and 3.0 schemas](https://www.openapis.org/) for all Kubernetes API
+endpoints. OpenAPI v3 is the preferred method for accessing OpenAPI as it
+provides a more comprehensive and accurate view of the API. It includes all
+available API paths, as well as all resources consumed and produced for every
+operation on every endpoint. This also includes any extensibility components
+that a cluster supports. This document is significantly larger than Discovery,
+which is a lightweight alternative that provides a subset of information in the
+OpenAPI document.
+
+## Discovery API
+
+Kubernetes publishes a list of all group versions and resources supported via
+the Discovery API. This includes the following for each resource:
+
+- Name
+- Cluster or Namespace scope
+- Endpoint URL and supported verbs
+- Alternative names
+- Group, version, kind
+
+The API is available both aggregated and unaggregated form. The aggregated
+discovery serves two endpoints while the unaggregated discovery serves a
+separate endpoint for each group version.
 
-Consider using one of the [client libraries](/docs/reference/using-api/client-libraries/)
-if you are writing an application using the Kubernetes API.
+### Aggregated Discovery
+
+{{< feature-state state="beta"  for_k8s_version="v1.27" >}}
+
+Kubernetes offers beta support for aggregated discovery, publishing
+all resources supported by a cluster through two endpoints (`/api` and
+`/apis`). Requesting this
+endpoint drastically reduces the number of requests sent to fetch the
+discovery for the average Kubernetes cluster. This may be accessed by
+requesting the respective endpoints with an Accept header indicating
+the aggregated discovery resource:
+`Accept: application/json;v=v2beta1;g=apidiscovery.k8s.io;as=APIGroupDiscoveryList`.
+
+Without the Accept header resource type explicitly indicated, the default
+response for the `/api` and `/apis` endpoint is an unaggregated discovery
+document.
+
+The [discovery document](https://github.com/kubernetes/kubernetes/blob/master/api/discovery/aggregated_v2beta1.json) serving built-in resources can be found in the Kubernetes Github repository.
+
+The endpoint also supports ETag and protobuf encoding.
+
+### Unaggregated Discovery
```

Comment:
Why lowercase? I think referring to the API as proper noun (Discovery API) makes sense, so the two forms of the API (aggregated and unaggregated) should still have Discovery capitalized?

Reply:
- This will localize better if it's all lowercase. We're explaining the concept of _unaggregated discovery_, even if the API that enables this also contains the string `Discovery`
- It's not exactly a resource kind like ConfigMap or Deployment are; you can't `kubectl get Discovery` and have that do something helpful.
- we write Kubernetes-specific concepts in sentence case, and then italicize them for their first use.
  eg: A _ConfigMap_ is an object that stores values&hellip;
  eg: In Kubernetes, _eviction_ refers to&hellip;

If we were to write something in UpperCamelCase, it could be **APIGroupDiscovery**; however, I don't think that particularly helps either. So, I recommend “discovery” here.

Reply:
Fixed.